src/main/java/com/google/sps/data/ScheduledInterviewRequest.java

```diff
@@ -62,6 +65,10 @@ public boolean getHasStarted() {
     return hasStarted;
   }
 
+  public String getShadow() {
+    return shadow;
+  }
+
   public boolean equals(Object o) {
```

Comment:
Can we get rid of this function? Or at least make sure you're using .equals() for Shadow because it's a String?